examples/post_training_quantization/torch/mobilenet_v2/main.py

```diff
@@ -0,0 +1,179 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import re
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+import nncf
+import numpy as np
+import openvino.runtime as ov
+import torch
+from sklearn.metrics import accuracy_score
+from openvino.offline_transformations import compress_quantize_weights_transformation
+from torchvision import datasets
+from torchvision import models
+from torchvision import transforms
+from tqdm import tqdm
+
+FOOD101_CLASSES = 101
+ROOT = Path(__file__).parent.resolve()
+DATASET_PATH = Path(sys.argv[1]) if len(sys.argv) > 1 else ROOT / 'dataset'
+CHECKPOINT_URL = 'https://huggingface.co/AlexKoff88/mobilenet_v2_food101/resolve/main/pytorch_model.bin'
+
+
+def fix_names(state_dict):
```

Comment:
This is not clear why we should call this function. Maybe it is better to update the checkpoint itself and completely remove this function?